cobigen/index.asciidoc

```diff
@@ -1,28 +1,30 @@
-= CobiGen Cli
+= Tutorial for Java Backend Generation with CobiGen
 ====
-CobiGen is a generic incremental code generator. It allows you build Java CRUD application based on the devonfw architecture including all software layers. You can generate all necessary classes and services (DAOs, Transfer Objects, simple CRUD use cases with REST services) of the application from one single entity class.
+CobiGen is a generic incremental code generator. It allows you to build Java CRUD application based on the devonfw architecture including all software layers. You can generate all necessary classes and services (DAOs, Transfer Objects, simple CRUD use cases with REST services) of the application from one single entity class.
 
 CobiGen provides Maven and Eclipse integrations. But there is also a command line interface (CLI) which enables the generation of code using only few commands. In this scenario, you will learn how to install and use the cobigen cli inside a java project.
 
 More information about CobiGen on https://devonfw.com/website/pages/docs/master-cobigen.asciidoc.html
 ====
 
-To use the cobigen cli you first have to install the devonfw ide. You will find more information about devonfw on https://devonfw.com/website/pages/welcome/welcome.html.
 [step]
 --
-installDevonfwIde(["java","mvn"])
+restoreDevonfwIde(["java","mvn"])
 --
 
-Now, you have to download cobigen
+For this tutorial an installation of the Devon IDE is required, which was already set up for you.
+To begin you need to install Cobigen and create a Project.
 [step]
+== Install CobiGen and create Java Project
 --
 installCobiGen()
+createDevon4jProject("cobigenexample")
 --
 
-Now you are ready to create your first project
+Now Build the java project
 [step]
 --
-createDevon4jProject("cobigenexample")
+buildJava("cobigenexample", false)
```

Comment:
I'm not sure, but I think you need to call buildJava after creating the entity class for CobiGen to work properly